docs/todos.md

```diff
@@ -26,10 +26,10 @@ If you want to see TODOs as part of `ember-template-lint`'s output, you can incl
 ember-template-lint . --include-todo
 ```
 
-If an error is fixed manually, `ember-template-lint` will let you know that there's an outstanding TODO file. You can remove this file by running `--fix`
+If an error is fixed manually, `ember-template-lint` will let you know that there's an outstanding TODO file. You can remove this file by running `--update-todo`
 
 ```bash
-ember-template-lint . --fix
+ember-template-lint . --update-todo
```

Comment:
The original content is correct, though perhaps the prose leading up to this is a bit confusing.

This is specifically talking about the following:

* you have a pre-existing issue in a template, that you had previously ran `--update-todo` for
* you **fix** that issue (no longer requiring the todo file)
* at this point, running `ember-template-lint` on that file will error (because there is a stale todo that needs to be cleaned up)
* running `ember-template-lint --fix` will do that cleanup for you

Reply:
@rwjblue 2 concerns: 
1. see the screenshot that I posted - it explicitly instructs the user to run `--update-todo` to remove the stale todo, hence the PR. I don't know how much more proof I could provide here. 

2. This is perhaps orthogonal to this PR but related in principle: `--fix` is already used to apply auto fixes other things; I had planned to raise this concern even before I discovered that the documentation was incorrect. What happens if I don't want it to also auto-fix everything else?

Reply:
You _can_ use `--update-todo` that is definitely fine. But `--fix` is a much more preferred here generally, using `--update-todo` will force **every** current violation to be migrated to a todo whereas `--fix` will only ever remove a todo that is no longer needed.

Also, it should *always* be safe to run `--fix`. If it is not in some circumstance, please open an issue dedicated to that scenario.

Reply:
We should definitely update the error message. That was written before we implement the auto-fix capability. 

Also was going to mention what Rob said about the usage of `--fix`. Why would you ever not want `--fix` to fix all the things that can be fixed? I also believe, as @rwjblue said, it should always be safe to run this.

I think the action item from this is to update the error message.